CHANGELOG.md

```diff
@@ -2,14 +2,28 @@
 
 All notable changes to this project will be documented in this file.
 
-The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
+The format is based on [Keep a Changelog](http://keepachangelog.com/) and this
+project adheres to [Semantic Versioning](http://semver.org/).
 
-## [v.1.6.0]
+## [Unreleased]
+
+## [v.1.7.0]
+
+### BREAKING CHANGES
 
-## BREAKING CHANGES
+- Now uses Node version 12.18.1
```

Comment:
This.... is probably fine, but we're going to need to have a more explicit warning to Core users when we put the PR in to update the default deployment.